### ISSUE TITLE
HTML Cache META attributes doesn't work

### ISSUE THREAD

**Issue by jo2010** [google/web-starter-kit#924]
Even with the following in the HTML files, it is cached. I think, the service-worker.js is causing this issue.

  <meta http-equiv="Cache-Control" content="no-cache, no-store, must-revalidate"/>
    <meta http-equiv="expires" content="Tue, 01 Jan 1980 1:00:00 GMT" />
    <meta http-equiv="pragma" content="no-cache" />
    <META HTTP-EQUIV="CACHE-CONTROL" CONTENT="NO-CACHE">

**Comment by Garbee** [comment]
Yup, SW overrides the network cache. You need to handle this in your applications Service Worker or remove the SW if you don't want to handle the client-side application caching manually.

**Comment by jo2010** [comment]
@Garbee Is there a way I can exclude certain html files from the caching?

**Comment by Montesdodgers13** [comment]
My HTML cache attributes still doesn't work any comments or update
